src/models/BibDetails.ts

```diff
@@ -43,6 +45,17 @@ export default class BibDetails {
     this.bottomDetails = this.buildBottomDetails()
   }
 
+  buildOwner(): string[] {
```

Comment:
Just curious, why does this have to be an array if it always returns one element?

Reply:
I think I was trying to keep the details as standard as possible. The subject heading urls are nested arrays, and the intention was to have all the values for the bib details be string arrays.